packages/tests/tests/store/error_handling.node.spec.ts

```diff
@@ -60,7 +60,7 @@ describe("Waku Store, error handling", function () {
         TestDecoder,
         customDecoder1
       ])) {
-        msgPromises;
+        await Promise.all(msgPromises);
```

Comment:
I see we are awaiting for promises, instead of just ignoring them (before)

What changed with this PR that caused this?